openaev-api/src/main/java/io/openaev/rest/stream/StreamApi.java

```diff
@@ -64,10 +66,16 @@ private void sendStreamEvent(FluxSink<Object> flux, BaseEvent event) {
     flux.next(message);
   }
 
+  private static final EnumSet<ResourceType> RESOURCES_STREAM_BLACKLIST =
+      EnumSet.of(ResourceType.CVE, ResourceType.PAYLOAD);
+
   @Async
   @Transactional
   @TransactionalEventListener
   public void listenDatabaseUpdate(BaseEvent event) {
+    if (RESOURCES_STREAM_BLACKLIST.contains(event.getInstance().getResourceType())) {
```

Comment:
```suggestion
    if (event.isListened()) {
```
*Question :* Wouldn't it be better to rely on the isListened and override it in the Java POJO to avoid having multiple ways of disabling listening of events ?